compiler/env/OMRClassEnv.hpp

```diff
@@ -122,6 +124,20 @@ class OMR_EXTENSIBLE ClassEnv
 
    bool classUnloadAssumptionNeedsRelocation(TR::Compilation *comp);
 
+   /** \brief
+    *	    Populates a TypeLayout object.
+    *
+    *  \param region
+    *     The region used to allocate TypeLayout.
+    * 
+    *  \param opaqueClazz
+    *     Class of the type whose layout needs to be populated.
+    * 
+    *  \return
+    *     Returns a NULL pointer.
+    */
+   TR::TypeLayout* enumerateFields(TR::Region& region, TR_OpaqueClassBlock * clazz, TR::Compilation *comp) { return NULL; }
```

Comment:
I'm wondering if the return type should be `const TR::TypeLayout*`. It would require using `const` in a few other places, but that should be ok since the type layout can never change anyways.

Reply:
Yes it should. I have added `const` in.